samples/03_org_administrators/org_wide_dependency_graph.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
```

Comment:
<p>Might help to add a line or so about <code>.gml</code> files for the beginner user, or link to some resource to learn more.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2311/discussion/#comment-2193902650'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>